xml/admin_monitoring_alerting.xml

```diff
@@ -1071,30 +1071,26 @@ monitoring:
        <term><option>MON_DOWN</option></term>
        <listitem>
          <para>One or more monitor daemons is currently down.
-           The cluster requires a majority (more than 1/2) of
-           the monitors in order to function. When one or more
-           monitors are down, clients may have a harder time
-           forming their initial connection to the cluster as
-           they may need to try more addresses before they
-           reach an operating monitor.</para>
-         <para>The down monitor daemon should generally be
-           restarted as soon as possible to reduce the risk
-           of a subsequen monitor failure leading to a service outage.</para>
+           The cluster requires a majority of the monitors in
+           order to function. When one or more
+           monitors are down, clients have a harder time
+           forming an initial connection to the cluster.</para>
+         <para>Restart the down monitor daemon as soon as possible to reduce the risk
```

Comment:
1. Is "down monitor daemon" using _down_ as an adjective? It seems ambiguous. Would "the monitor daemon that is down" or some other description such as "the failed monitor daemon" be better?

2. I think we usually have [para] [/para] tags on their own lines.